.github/workflows/container.yml

```diff
@@ -3,8 +3,22 @@ name: Container
 on:
   push:
     branches: [ main ]
+    paths:
+      - '**.go'
+      - 'go.mod'
+      - 'go.sum'
+      - Dockerfile*
+      - .dockerignore
+      - .github/workflows/container.yml
   pull_request:
     branches: [ main ]
+    paths:
+      - '**.go'
+      - 'go.mod'
+      - 'go.sum'
+      - Dockerfile*
+      - .dockerignore
+      - .github/workflows/container.yml
```

Comment:
These paths look good to me, you might need to had:

* the Rust files (.rs, cargo, rust-toolchain.toml) for the BPF program
* the `3rdparty` directory for the libbpf Git submodule
* the Makefile